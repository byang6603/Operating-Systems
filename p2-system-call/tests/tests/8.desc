Exec Test: Verify that exec() resets crash flag for new program.
