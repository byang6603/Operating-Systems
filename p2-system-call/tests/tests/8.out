XV6_TEST_OUTPUT In exec program! Calling crash.
